docs/operator/quick-start.md

```diff
@@ -49,7 +45,7 @@ kubectl get configauditreports -o wide
 
 ```
 NAME                          SCANNER     AGE    CRITICAL  HIGH   MEDIUM   LOW
-replicaset-nginx-78449c65d4   Trivy   2m7s   0         0      6        7
+replicaset-nginx-78449c65d4   Trivy-Operator   2m7s   0         0      6        7
```

Comment:
it should be `Trivy` indeed, this was changed on https://github.com/aquasecurity/trivy-operator/pull/126